src/components/Card.vue

```diff
@@ -7,7 +7,7 @@
       <span>{{ title }}</span>
     </div>
     <div v-if="image" class="card__image">
-      <g-image :src="image" />
+      <g-image :src="image" aria-hidden="true" />
```

Comment:
This is the only change I'm not familiar with - is this because we don't have good alt text for the image?

Reply:
@TerabyteTiger  Yes, screenshots are of little use to blind people, and even with alt , alt does not describe a complex screenshot well. This element there instead becomes an obstacle to their operation. This property hides this element for them.

Reply:
Thank you for the explanation! Changes look good to me 👍🏻